### ISSUE TITLE
Add product from Sentinel-1 Amazonas product - BIOMASS theme

### ISSUE THREAD

**Issue by aapopescu** [eurodatacube/eodash#2411]
https://sen4ama.gisat.cz/ 

https://github.com/gisat/s14amazonas 

contact: https://github.com/sivasanarul 

@AlessandroScremin cc: @dmoglioni 

-> to be included in trilateral dashboard

**Comment by lubojr** [comment]
Currently the first draft of sen4ama Amazonas integration is on https://eodash-trilateral-testing.eox.at/amazonas-sen4ama/
The visualisation seems fine and working to me but maybe we could also show the origina S1 14 day composite that it was based on? Up to decide. In that case evalscript color would need to be changed from grayscale to some other color to contrast with S1 better. 

Please have a look @AlessandroScremin @aapopescu and suggest if leaving like this is OK or we should reference other layers as well.

**Comment by santilland** [comment]
@santilland check if we can do requests from monthly steps if SH can handle the response.
The data seems to be as expected when comparing to original

**Comment by AlessandroScremin** [comment]
@santilland 
color changed to RED.

**Comment by lubojr** [comment]
> * [x]  @santilland check if we can do requests from monthly steps if SH can handle the response.
> * [x]  The data seems to be as expected when comparing to original.
> * [x]  S1 display not needed for now, set S2 cloudless as default backgorund.
> * [x]  @AlessandroScremin to change color to red

All integrated.

- [ ] We are just missing a markdown description and we are good to merge to be included in the biomass story. @AlessandroScremin @aapopescu

**Comment by lubojr** [comment]
@AlessandroScremin Are there any updates on the markdown description for Sen4Ama data?

**Comment by aapopescu** [comment]
markdown is ready

**Comment by lubojr** [comment]
Thank you.
Prepared https://github.com/eurodatacube/eodash/pull/2505 to staging and created STAC catalog reference YAML and copied assets.
Closing the issue as done.

**Comment by lubojr** [comment]
@aapopescu PR merged to staging.

An attempt for a standard custom area chart integration is now in a branch here: https://eodash-trilateral-testing.eox.at/amazonas-sen4ama-chart/?indicator=sen4ama

**Comment by lubojr** [comment]
Nevermind, with @santilland we agreed on an approach of time series of "detections pixels vs nodata pixels". I will redo the chart interpretation. Reopening the issue

**Comment by AlessandroScremin** [comment]
@lubojr 

I was looking at the chart...but it doesn't seems much useful with all values set to 1.

The map is a binary mask no data value = 0) and 1. the idea was to compute  the bumber of pixels with value 1 in the custom AOI and compute the area knowing the resolution of the pixel and the number of pixels 

what I see in the dashboard is this:

![image](https://github.com/eurodatacube/eodash/assets/65602207/fd195f57-b776-4947-8ac3-82da8b11e1b3)

**Comment by lubojr** [comment]
> Nevermind, with @santilland we agreed on an approach of time series of "detections pixels vs nodata pixels". I will redo the chart interpretation. Reopening the issue

@AlessandroScremin Yes. In the last comment I confirm that the current implementation is not final. We will change it in the nearby future (using the same principle as for WSF dataset)

**Comment by aapopescu** [comment]
@lubojr, any info missing here?

**Comment by lubojr** [comment]
No, we are not missing any info, it is on our end. We need to implement the different handling of the chart for this dataset. Shared implementation with WSF.

**Comment by lubojr** [comment]
@lubojr Add Sentinel Cloudless Layer as a default basemap. Think of a better way how to call the statistics endpoint for better display.

**Comment by lubojr** [comment]
moved to https://github.com/ESA-eodashboards/eodashboard-catalog/issues/117
